chore: Add troubleshooting entry for Fusion non-resync and env var workaround (#770)
seqeralabs/docs: platform-cloud/docs/troubleshooting_and_faqs/studios_troubleshooting.md

This is displayed because logging is set to `stderr` by default to ensure all logs are shown during the session, and can safely be ignored.

## Running session does not show new data in object storage

> By default, Fusion does not resync objects from the remotely mounted data-link(s) after initial mounting. This means that when running a session with data mounted, if the underlying storage updates, the data is not re-synced to the studio session. This behavior can be changed by setting the `FUSION_REFRESH_TIMEOUT` environment variable in your studio session to a number of seconds. This will force Fusion to refresh the view of the mounted data-link(s).

Reviewer: Setting the env "in your studio session" may be misleading. Setting the env var inside a running session with "export FUSION_REFRESH_TIMEOUT" won't change the behaviour of the outer fusion session. My understanding is that it needs to be set outside of the fusion process (so it can be inherited by fusion). Setting `FUSION_REFRESH_TIMEOUT` in the Compute Environment, or as a variable while configuring the Studio will work.

Author: > Setting FUSION_REFRESH_TIMEOUT in the Compute Environment

I don't think we want to recommend this approach, because of the potential unknown impact if a user inadvertently runs a pipeline in that CE. I'll update to be more explicit that it needs to be added as an env var when the session is added.

Author: Fixed, pushed, and re-review requested.

This is displayed because logging is set to `stderr` by default to ensure all logs are shown during the session, and can safely be ignored.

## Running session does not show new data in object storage

> By default, Fusion does not resync objects from the remotely mounted data-link(s) after initial mounting. This means that when running a session with data mounted, if the underlying storage updates, the data is not re-synced to the studio session. This behavior can be changed when [adding a studio session](../studios/managing#add-a-studio) by defining the `FUSION_REFRESH_TIMEOUT` environment variable to N number of seconds (e.g. `30`). This will force Fusion to refresh the view of the mounted data-link(s) every N seconds.

Reviewer 2: ```suggestion
By default, Fusion does not resync objects from remotely mounted data-link(s) after initial mounting. 

If you have a running session with data mounted and the underlying storage is updated, the data will not be resynced to the Studio session. 

You can change this behavior when you are [adding a Studio session](../studios/managing#add-a-studio) by defining the `FUSION_REFRESH_TIMEOUT` environment variable to a specified number of seconds (e.g., `30`). This will force Fusion to refresh the view of the mounted data-link(s) at the specified interval.
```
